sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -168,18 +169,22 @@ def run_job(self):
         course = self.job_details['course']
         gradeable = self.job_details['gradeable']
         config_id = self.job_details['config_id']
+        config_data = self.job_details['config_data']
 
         # error checking
         # prevent against backwards crawling
-        if '..' in semester or '..' in course or '..' in gradeable or '..' in config_id:
```

Comment:
You're removing the `'..'` check here as `config_id` should only be of type `int`. It would be good to add a type check to that effect for `config_id`, in the case someone figures out how to dump arbitrary files into the job directory, they could make `config_id` whatever type or value they wanted.